content/de/faq.md

```diff
@@ -1,34 +1,32 @@
 
 ## Version 1.0.x?
 
-io.js has moved to [Semver](http://semver.org/) and the changes between Node.js™ 0.10 and io.js 1.0.0 were significant enough
-to warrant a major version increment.
+io.js verwendet die [Semver](http://semver.org/)-Versionierung und die Änderungen zwischen Node.js™ 0.10 und io.js 1.0.0 waren gravierend genug, einen großen Versionssprung zu rechtfertigen.
 
-Our [CHANGELOG](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) for v1.x provides a [summary of changes from Node.js v0.10.35 to io.js v1.0.x](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#summary-of-changes-from-nodejs-v01035-to-iojs-v100).
+Unser [CHANGELOG](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) für v1.x enthält eine [Übersicht der Änderungen von Node.js v0.10.35 zu io.js v1.0.x](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md#summary-of-changes-from-nodejs-v01035-to-iojs-v100).
 
-## How can I contribute?
+## Wie kann ich mitmachen?
 
-Everyone can help. io.js adheres to a [code of conduct](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), and contributions, releases, and contributorship are under an [open governance](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme) model.
+Jeder kann mithelfen. io.js folgt einem [Verhaltenscodex](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-of-conduct), und alle Beiträge und Veröffentlichungen stehen unter einem [offenen Aufsichts-Modell](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme).
 
-To get started, there are open [ discussions on GitHub](https://github.com/iojs/io.js/issues), and we'd love to hear your feedback.
-Becoming involved in discussions is a good way to get a feel of where you can help out further. If there is
-something there you feel you can tackle, please [make a pull request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
+Um direkt einzusteigen, schaue dir die offenen [Diskussionen auf GitHub](https://github.com/iojs/io.js/issues) an, wir würden uns über dein Feedback freuen.
+Sich an Diskussionen zu beteiligen ist ein guter Weg, ein Gespür dafür zu bekommen, wo du weiterhelfen kannst. Wenn es etwas gibt, dass du verändern möchtest, schicke bitte einen [Pull Request](https://github.com/iojs/io.js/blob/v1.x/CONTRIBUTING.md#code-contributions).
 
-In addition, using [Nodebug.me](http://nodebug.me/) is a good way to help Triage the issues in the backlog.
+Zusätzlich gibt es mit [Nodebug.me](http://nodebug.me/) eine Möglichkeit dabei zu helfen, bestehende Themen aus dem Backlog zu sichten.
 
-## Where do discussions take place?
+## Wo finden Diskussionen statt?
 
-There is an #io.js channel on Freenode IRC. We keep logs of the channel [here](http://logs.libuv.org/io.js/latest).
+Es gibt einen #io.js-Kanal auf Freenode IRC. Diskussionslogs dieses Kanals gibt es [hier](http://logs.libuv.org/io.js/latest).
 
-## What is open source governance?
+## Was ist Open-Source Governance?
```

Comment:
I wasn't sure if this is really the best translation. Even the Wikipedia uses "Open Governance" as fixed term in the German article.